curriculum/challenges/english/01-responsive-web-design/applied-visual-design/create-movement-using-css-animation.english.md

```diff
@@ -41,11 +41,11 @@ Add a horizontal motion to the <code>div</code> animation. Using the <code>left<
 ```yml
 tests:
   - text: The <code>@keyframes</code> rule for <code>0%</code> should use the <code>left</code> offset of 0px.
-    testString: assert(code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?left:\s*?0(px)?;\s*?}/gi));
+    testString: assert(code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?left:\s*?0(px)?;\s*?}/gi) || code.match(/0%\s*?{\s*?background-color:\s*?blue;\s*?left:\s*?0(px)?;\s*?top:\s*?0(px)?;\s*?}/gi) || code.match(/0%\s*?{\s*?left:\s*?0(px)?;\s*?background-color:\s*?blue;\s*?top:\s*?0(px)?;\s*?}/gi));
   - text: The <code>@keyframes</code> rule for <code>50%</code> should use the <code>left</code> offset of 25px.
-    testString: assert(code.match(/50%\s*?{\s*?background-color:\s*?green;\s*?top:\s*?50px;\s*?left:\s*?25px;\s*?}/gi));
+    testString: assert(code.match(/50%\s*?{\s*?background-color:\s*?green;\s*?top:\s*?50px;\s*?left:\s*?25px;\s*?}/gi) || code.match(/50%\s*?{\s*?left:\s*?25px;\s*?background-color:\s*?green;\s*?top:\s*?50px;\s*?}/gi) || code.match(/50%\s*?{\s*?background-color:\s*?green;\s*?left:\s*?25px;\s*?top:\s*?50px;\s*?}/gi));
   - text: The <code>@keyframes</code> rule for <code>100%</code> should use the <code>left</code> offset of -25px.
-    testString: assert(code.match(/100%\s*?{\s*?background-color:\s*?yellow;\s*?top:\s*?0(px)?;\s*?left:\s*?-25px;\s*?}/gi));
+    testString: assert(code.match(/100%\s*?{\s*?background-color:\s*?yellow;\s*?top:\s*?0(px)?;\s*?left:\s*?-25px;\s*?}/gi) || code.match(/100%\s*?{\s*?left:\s*?-25px;\s*?background-color:\s*?yellow;\s*?top:\s*?0(px)?;\s*?}/gi) || code.match(/100%\s*?{\s*?background-color:\s*?yellow;\s*?left:\s*?-25px;\s*?top:\s*?0(px)?;\s*?}/gi));
```

Comment:
I have tested the below regex that is much simpler. Please click the Commit suggestion if you agree.
```suggestion
    testString: assert(code.match(/100%\s*?{[\s\S]*?left:\s*?-25px(;[\s\S]*?|\s*?)}/gi));
```